src/Compiler/RefC/CC.idr

```diff
@@ -41,6 +43,14 @@ findLDFLAGS
          | Just ldflags => pure ldflags
        pure ""
 
+findLDLIBS : IO String
+findLDLIBS
+  = do Nothing <- getEnv "IDRIS2_LDLIBS"
```

Comment:
It should be `idrisGetEnv` instead of `getEnv` in the whole file, because this function checks for the presence of used environment variables in the help description.

Reply:
Done. It does introduce a dependency on `Idris.Env`, but I think that's acceptable.